app/views/layouts/_navbar.html.haml

```diff
@@ -19,7 +19,7 @@
           - if current_user.proposals.any?
             = render partial: "layouts/nav/proposals_link"
 
-          - if current_event && current_user.reviewer_for_event?(current_event)
+          - if current_event && current_user.reviewer_for_event?(current_event) || current_event && current_user.program_team_for_event?(current_event)
```

Comment:
This can factor into current_event && (current_user.reviewer_for_event?(current_event) || current_user.program_team_for_event?(current_event))